modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - recurring_events
+id: eventinstance.field_ls_image
+field_name: field_ls_image
+entity_type: eventinstance
+type: entity_reference
+settings:
+  target_type: media
```

Comment:
We need to limit it to Image bundle only

Reply:
Already limited here - https://github.com/fivejars/openy_gated_content/pull/30/files#diff-890ad3e5720a1a271a5ff4539678c6ddR28

Reply:
Looks like there is a problem with Open Y entity browser views (it should be limited by image type)